earthpy/io.py

```diff
@@ -265,9 +279,12 @@ def _download(self, url, path, kind, replace, verbose):
         path = op.expanduser(path)
         if replace is False and op.exists(path):
             return path
-
-        if verbose is True:
-            print("Downloading from {}".format(url))
+        # We know here the level is being set to INFO but it is still not printing (as a log) in the terminal
+        if verbose:
```

Comment:
@sgillies if you have time and patience we are struggling a bit with logging. In this example, we are printing out the level to ensure that the level is "INFO". however even with it being set to INFO, it does NOT seem to print the log out for the user. do you happen to know why this might be happening? it DOES work if we set the logger to INFO at the root using `logging.basicConfig()` 

Any thoughts on this?